misaligned.cpp

```diff
@@ -1,21 +1,36 @@
 #include <iostream>
 #include <assert.h>
+#include <string.h>
+
+using namespace std;
+
+const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
+const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+char previousMinorColor[] = "defaultMinor";
 
 int printColorMap() {
-    const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
-    const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
-    int i = 0, j = 0;
-    for(i = 0; i < 5; i++) {
-        for(j = 0; j < 5; j++) {
-            std::cout << i * 5 + j << " | " << majorColor[i] << " | " << minorColor[i] << "\n";
-        }
+  int i = 0, j = 0;
+  for(i = 0; i < 5; i++) {

```

Comment:
use of magic number 5 ! It would be difficult for people maintaining this code later to understand why 5 was chosen!